### PR TITLE
Allow policy_arns to be attached, needed for AWS managed policies

### PR DESCRIPTION
**Pull request by msooszalando**
This change is needed to correctly support AWS managed policies, see https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_managed-vs-inline.html?shortFooter=true#aws-managed-policies for such policies. They are very useful and lower the burden of maintenance of policy documents.

### CODE REVIEW

**Comment by szuecs** [comment]
@msooszalando please add your signature in the comment of your PR `git commit -s` and rebase to one commit.

Why do we need to change sevenseconds and can't just change our awsconfig and rollout the change with sevenseconds?

@sarnowski is this the change you talked about with me some days ago?

File: sevenseconds/config/iam.py [hunk 1]
```diff
@@ -44,12 +44,16 @@ def configure_iam_policy(account: object):
             with ActionOnExit('Checking role {role_name}..', **vars()) as act:
                 try:
                     role = iam.Role(role_name)
+                    # TODO make policy optional
```
Comment (szuecs) [inline sevenseconds/config/iam.py:47]: Can you fix the TODO, before the merge?

**Review by szuecs — CHANGES_REQUESTED** [review]
the TODO needs to be fixed

**Comment by mikkeloscar** [comment]
Managed policies are now supported since #108 Closing this!

**Comment by msooszalando** [comment]
Woooow! Yaaaay! :heart_eyes:

**Pull request closed without merging**